exchanges/credentials.go

```diff
@@ -289,12 +125,14 @@ func (b *Base) GetDefaultCredentials() *Credentials {
 
 // GetCredentials checks and validates current credentials, context credentials
 // override default credentials, if no credentials found, will return an error.
-func (b *Base) GetCredentials(ctx context.Context) (*Credentials, error) {
-	value := ctx.Value(contextCredentialsFlag)
+func (b *Base) GetCredentials(ctx context.Context) (*account.Credentials, error) {
+	value := ctx.Value(account.ContextCredentialsFlag)
 	if value != nil {
-		ctxCredStore, ok := value.(*contextCredentialsStore)
+		ctxCredStore, ok := value.(*account.ContextCredentialsStore)
 		if !ok {
-			return &Credentials{}, errContextCredentialsFailure
+			// NOTE: Return empty credentials on error to limit panic on
+			// websocket handling.
```

Comment:
Could you elaborate on this one? I'm only seeing four instances of `GetCredentials` being called in websockets and when it does err, it does not use an `account.Credentials`

Reply:
![image](https://user-images.githubusercontent.com/9391715/173000192-3b03ce2d-e5d4-418e-a1e3-e54befead05a.png)